File: Java/src/listados/listado2/Ejercicio19.java
package listados.listado2;
import java.util.Scanner;

/**
* Escribe un programa que pida un número entero entre uno y doce e imprima el número de 
* días que tiene el mes correspondiente
*
* @author Fco Javier Frias Serrano
* @version 1.0
*/

class Ejercicio19{
	
	public static void main(String[] args){
		
	int num;
	
	Scanner s = new Scanner(System.in);
	
	System.out.println("Introduzca el numero del mes y le digo cuantos dias tiene: ");
	num=s.nextInt();
	
	switch (num) {
	case 1:
	case 3:
	case 5:
	case 7:
	case 9:
	case 11:
		System.out.println("El mes correspondiente tiene 31 dias.");	
		break;
	case 4:
	case 6:
	case 8:
	case 10:
	case 12:
		System.out.println("El mes correspondiente tiene 30 dias.");
		break;
	case 2:
		System.out.println("El mes correspondiente tiene 28 dias.");
		break;
	default:
		System.out.println("Ese mes no existe");
		break;
	}
	
	}
	
}

File: Python/alternativa/Ejercicio19.py
'''
Escribe un programa que pida un número entero entre uno y doce e imprima el número de días que tiene el mes correspondiente
 
@author: Fco Javier Frias Serrano
'''
    
print("Introduzca el numero del mes y le digo cuantos dias tiene: ")
num=int(input())
    
if num==1 or num==3 or num==5 or num==7 or num==9 or num==11:
    print("El mes correspondiente tiene 31 dias")   
elif num==4 or num==6 or num==8 or num==10 or num==12:
    print("El mes correspondiente tiene 30 dias")
elif num==2:
    print("El mes correspondiente tiene 28 dias")
else:   
    print("Ese mes no existe")
